bot/cogs/verification.py

```diff
@@ -27,12 +29,16 @@
 If you'd like to unsubscribe from the announcement notifications, simply send `!unsubscribe` to <#{Channels.bot}>.
 """
 
+PERIODIC_PING = (f"@everyone To verify that you have read our rules, please type `!accept`."
```

Comment:
this doesn't have to be an f-string.

We also prefer that you use K&R indentation style for these, which would look like this:
```py
PERIODIC_PING = (
    "first string"
    f"other string"
)
```